src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -231,6 +231,42 @@ public bool IsSetAsInput
             }
         }
 
+        /// <summary>
+        /// Output nodes are used applications that consume graphs outside of Dynamo such as Optioneering, Optimization, 
+        /// and Dynamo Player. Output nodes can be any node that returns a single output or a dictionary.  
```

Comment:
this is not really true though, what about a codeblock that only returns one output port?

Reply:
also for codeblock, you could just put everything in a list - right?

Reply:
perhaps.... The Is Input is side is clear because the node can switch to having inputs to not having them based on the DS.  But does this feel like a node that returns a list.  Usually most nodes don't return lists that are not homogeneous by type.  This feels more like it should be put in a dictionary.

Reply:
For now I update the comment but leave the behavior.  Code block nodes will not have outputs for now even it is just a matter of symmetrical clarity.